### ISSUE TITLE
Bring input[type=range] inline with other inputs while focused

### ISSUE THREAD

**Issue by mouse0270** [foundryvtt/foundryvtt#9669]
### User Experience

Is their any particular reason, `input[type=range]` does not have a focus state like every other inputs, maybe something like
```css
input[type=range]:focus::-webkit-slider-runnable-track {
    box-shadow: 0 0 5px var(--color-shadow-primary);
}
input[type=range]:focus::-webkit-slider-thumb {
    box-shadow: 0 0 5px var(--color-shadow-primary);
}
input[type=range]:focus::-moz-range-track {
    box-shadow: 0 0 5px var(--color-shadow-primary);
}
input[type=range]:focus::-moz-range-thumb {
    box-shadow: 0 0 5px var(--color-shadow-primary);
}
```
Its rather odd when tabbing through the controls and not being able to identify that your are currently focus on the `input[type=range]`. There are other steps such as using outline instead of box-shadow for accessibility reasons, but I feel like the input should at least replicate your other inputs at the bare minimum.

An example using these values:
![image](https://github.com/foundryvtt/foundryvtt/assets/564874/74c38e18-e594-4a96-b54a-86b5d0cb4321)
